nmstatectl/nmstatectl.py

```diff
@@ -394,3 +396,15 @@ def _filter_routes(state, patterns):
                 if fnmatch.fnmatch(route[Route.NEXT_HOP_INTERFACE], pattern):
                     routes[route_type].append(route)
     return routes
+
+
+def _filter_route_rule(state, patterns):
+
+    route_rules = {RouteRule.CONFIG: []}
+
+    for rule in state.get(RouteRule.CONFIG, []):
+        for pattern in patterns:
+            if fnmatch.fnmatch(rule, pattern):
```

Comment:
A rule is pointing to a route table, route table contains route next hop to interface.

`fnmatch.fnmatch(rule, pattern)` will always return False.

Reply:
@cathay4t 
I'll make the changes. I couldn't find example related or including route rule. Does the route table contains only next hop interface?